### DISCUSSION TITLE
"zoom" tool in internal obs browser

### DISCUSSION THREAD

**Discussion by theviper22** [obsproject/obs-studio#7454] [Development Questions] [answered]
The "zoom" tool in the internal browser of obs would be very useful, many pages have very small letters

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a feature request, and we are not currently accepting feature requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance, or post your suggestion on our [ideas page](https://ideas.obsproject.com)

Thank you!
